source/auditing-logging.txt

```diff
@@ -12,17 +12,77 @@ Auditing and Logging
    :depth: 1
    :class: onecol
 
-Intro statement
+To monitor and log |service| platform activities, use auditing and logs.
 
 {+service+} Features and Best Practices for Auditing and Logging
 ----------------------------------------------------------------
 
-Content here
+Auditing
+~~~~~~~~
+
+Database auditing lets administrators track system activity for 
+deployments with multiple users. |service| administrators can select 
+the actions, database users, |service| roles, and LDAP groups that they 
+want to audit. You can configure manual :manual:`auditing </core/auditing>` 
+in |service| of most of the documented :manual:`system event actions 
+<audit-action-details-results>`.
+
+You can :atlas:`set up database auditing <set-up-database-auditing>`.
+
+To learn about best practices for auditing the actions of temporary
+database users, see :atlas:`Audit Temporary Database Users <audit-temp-db-users>`.
+
+By default, |service| performs database authentication auditing in M10+ clusters
+to record authentication events, which include:
+
+- database users
+- source IP addresses
+- timestamps for successful and failed attempts.
```

Comment:
Database users and IP addresses are not, themselves, authentication events, right? There are authentication events which pertain to them, though. Are the recorded events database user login attempts and connection attempts from IP addresses? I think we should either stick with the characterization 'events' and then find a way to rewrite these items to be events, or we rework the wording above to say something like 'record authentication events, including those pertaining to:'